test/start_time_test.go

```diff
@@ -28,10 +28,10 @@ import (
 
 // TestStartTime tests that step start times are reported accurately.
 //
-// It runs a TaskRun with 5 steps that each sleep 10 seconds, then checks that
-// the reported step start times are 10+ seconds apart from each other.
+// It runs a TaskRun with 5 steps that each sleep 2 seconds, then checks that
+// the reported step start times are 2+ seconds apart from each other.
 // Scheduling and reporting specifics can result in start times being reported
-// more than 10s apart, but they shouldn't be less than 10s apart.
+// more than 2s apart, but they shouldn't be less than 2s apart.
```

Comment:
similar nit - keeping the specific numbers out of the commit is a bit easier than having to keep it in sync - and also it can be nice to try to explain where "2" is coming from

Reply:
Updated the comment to remove specific numbers and describe the tradeoff involved.